### ISSUE TITLE
[-] Installation failed!

### ISSUE THREAD

**Issue by yanisre** [b31ngD3v/bomberthon#9]
![image](https://user-images.githubusercontent.com/45793246/111051753-c5c46080-8455-11eb-8dcb-28b824b1f17b.png)

**Comment by yanisre** [comment]
i put my correct username and password but doesn't work

**Comment by razormaster** [comment]
As far as i understand, it doesn't ask you your own github account but the author itself put a self credentials on its scripts to be unable to use from other script users, too bad..

**Comment by b31ngD3v** [comment]
Sorry, But I've no clue what is going on.

**Comment by razormaster** [comment]
> Sorry, But I've no clue what is going on.

can we talk on IG? to fix the error? 
![175200043_143589164311507_6340594995744460346_n](https://user-images.githubusercontent.com/82802705/115219280-f3c65000-a139-11eb-84de-997000328ddd.png)

**Comment by razormaster** [comment]
can we talk on IG? to fix the error?

we got this kind of error when it asked how many ..

![Uploading 175200043_143589164311507_6340594995744460346_n.png…]()

**Comment by b31ngD3v** [comment]
![](https://user-images.githubusercontent.com/64733912/113488411-94323880-94db-11eb-9f0b-dc05ea597d38.png)

**Comment by razormaster** [comment]
yes, i tried on windows as well and i got the said error, look to the screenshot that i sent to you mate.

**Comment by razormaster** [comment]
can we talk on IG or messenger or what IRC do you use? Signal?

**Comment by razormaster** [comment]
i tried on windows via command, it says the attack is on going but when i tried on my phone #, i didn't receive the spam sms.. what could be the problem though?

**Comment by b31ngD3v** [comment]
> yes, i tried on windows as well and i got the said error, look to the screenshot that i sent to you mate.

IG @b31ngD3v 
TG @b31ngD3v

**Comment by b31ngD3v** [comment]
> i tried on windows via command, it says the attack is on going but when i tried on my phone #, i didn't receive the spam sms.. what could be the problem though?

Country code ?

**Comment by razormaster** [comment]
It's +63 

(Philippines), would you mind if you can check the possible why it is not going to work?

**Comment by Nocturnal-Compiler** [comment]
it's because the chromedriver version is not updated, hence it cannot find the chrome binary files
 i will fix it in a bit
